rootston/desktop.c

```diff
@@ -203,16 +203,15 @@ struct roots_view *view_at(struct roots_desktop *desktop, double lx, double ly,
 struct roots_desktop *desktop_create(struct roots_server *server,
 		struct roots_config *config) {
 	struct roots_desktop *desktop = calloc(1, sizeof(struct roots_desktop));
+	assert(desktop);
```

Comment:
You shouldn't use assert for runtime errors.

Reply:
Okay. I've seen `assert` used to check allocations elsewhere in rootston's code - I'll remove them.